poll/public/html/poll_edit.html

```diff
@@ -1,75 +1,88 @@
-           <h2><label for="poll-feedback-editor">Feedback</label></h2>
-           <a href="//daringfireball.net/projects/markdown/syntax" target="_blank">Markdown Syntax</a> is supported.
-           <div id="poll-feedback-editor-container">
-             <textarea class="input setting-input" name="feedback" id="poll-feedback-editor"
-                       aria-describedby="poll-feedback-editor-help">{{feedback}}</textarea>
-           </div>
+            <h2><label for="poll-feedback-editor">{% trans 'Feedback' %}</label></h2>
+            {% blocktrans with link_start='<a href="//daringfireball.net/projects/markdown/syntax" target="_blank">' link_end='</a>' %}
+                {{link_start}}Markdown Syntax{{link_end}} is supported.
+            {% endblocktrans %}
+            <div id="poll-feedback-editor-container">
+                <textarea class="input setting-input" name="feedback" id="poll-feedback-editor"
+                          aria-describedby="poll-feedback-editor-help">{{feedback}}</textarea>
+            </div>
             <span class="tip setting-help" id="poll-feedback-editor-help">
-                This text will be displayed for the user as some extra feedback after they have
-                submitted their response to the poll.
+                {% blocktrans %}
+                    This text will be displayed for the user as some extra feedback after they have
+                    submitted their response to the poll.
+                {% endblocktrans %}
             </span>
         </li>
         <li class="field comp-setting-entry is-set">
             <div class="wrapper-comp-setting">
-                <label class="label setting-label poll-setting-label" for="poll-private-results">Private Results</label>
+                <label class="label setting-label poll-setting-label" for="poll-private-results">{% trans 'Private Results' %}</label>
                 <select id="poll-private-results" class="input setting-input" name="private_results"
                         aria-describedby="poll-private-results-help">
                     <!-- So far as I can see, there's not a proper style for checkboxes. LTI module does it this way. -->
-                    <option value="true" {% if private_results %} selected{% endif %}>True</option>
-                    <option value="false" {% if not private_results %} selected{% endif %}>False</option>
+                    <option value="true" {% if private_results %} selected{% endif %}>{% trans 'True' %}</option>
+                    <option value="false" {% if not private_results %} selected{% endif %}>{% trans 'False' %}</option>
                 </select>
             </div>
             <span class="tip setting-help" id="poll-private-results-help">
-                If this is set to True, don't display results of the poll to the user.
+                {% trans 'If this is set to True, don't display results of the poll to the user.' %}
```

Comment:
I believe this is where the error with the `{% trans %}` tag is happening - the apostrophe is ending the string prematurely, and the rest is getting passed as an argument to the tag. Try switching this particular string to be wrapped by double-quotes, and double-check the rest of the change for other instances of apostrophe use if that doesn't fix it.